src/main/java/com/safetynet/alerts/presentation/controller/MedicalRecordController.java

```diff
@@ -0,0 +1,162 @@
+package com.safetynet.alerts.presentation.controller;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.safetynet.alerts.configuration.DataConfig;
+import com.safetynet.alerts.data.io.JsonDAO;
+import com.safetynet.alerts.logging.LogHandlerTiny;
+import com.safetynet.alerts.logic.parsers.ModelObjectFinder;
+import com.safetynet.alerts.logic.service.MedicalRecordService;
+import com.safetynet.alerts.logic.updaters.ResultModel;
+import com.safetynet.alerts.logic.updaters.UpdateMedicalRecord;
+import com.safetynet.alerts.logic.JsonHandler;
+import com.safetynet.alerts.presentation.model.MedicalRecord;
+import com.safetynet.alerts.presentation.model.SafetyAlertsModel;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+/**
+ * RestController for /medicalRecord endpoint
+ *
+ * Includes POST/PUT/DELETE
+ */
+@RestController
+public class MedicalRecordController extends BaseController {
+
+    @Autowired
+    private MedicalRecordService medicalRecordService;
+
+    private String stringArrayToString(String[] stringArray) {
```

Comment:
this method seems used by several controllers, if it is the same implementation, move it to BaseController

Reply:
done, and also done with intArrayToStringArray as it is used for a similar purpose